client/styles/components/_overlay.scss

```diff
@@ -24,10 +24,9 @@
   display: flex;
   flex-wrap: wrap;
   flex-flow: column;
-  max-height: 80%;
```

Comment:
This is the style for ALL modal, not just "create collection", so we need to tread carefully here.  There are some larger modals like "keyboard shortcuts" where the `max-height` is required.  It makes it so that the modal itself is limited in height and forces the content section of the modal to have a scrollbar if the contents won't fit.

Reply:
I apologize for the oversight, I failed to consider the potential utilization of this class for other modals.

Reply:
@lindapaiste, we can solve this issue using the `:has()` pseudo-class. I have made the commit you can check it.